airgun/entities/sync_templates.py

```diff
@@ -0,0 +1,53 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep
+from airgun.navigation import navigator
+from airgun.views.template import SyncTemplatesView
+
+
+class SyncTemplatesEntity(BaseEntity):
+    endpoint_path = '/template_syncs'
+
+    def import_templates(self, values):
+        """Import Templates entity"""
+        view = self.navigate_to(self, 'import')
+        view.fill(values)
+        view.submit.click()
+
+    def export_templates(self, values):
+        """Export Templates entity"""
+        view = self.navigate_to(self, 'export')
+        view.fill(values)
+        view.submit.click()
+
+
+@navigator.register(SyncTemplatesEntity, 'All')
+class ShowMainPageActionNavigation(NavigateStep):
+    """Navigate to the page thats main Import Export Templates page"""
+    VIEW = SyncTemplatesView
+
+    def step(self, *args, **kwargs):
+        self.view.menu.select('Hosts', 'Sync Templates')
+
+
+@navigator.register(SyncTemplatesEntity, 'import')
```

Comment:
I'm not fan of these navigation endpoints. These are not two distinct pages, this is one page that shows or hides some fields depending on which radio value is selected in first field. This is what `ConditionalSwitchableView` widget is for, and that should be single navigation endpoint.

Reply:
Yeah , You are correct and thats the reason I had :

```
import_export = RadioGroup("//div[@class='col-md-6']")
```
in my code to implement ```ConditionalSwitchableView``` but not sure why I change my mind :)